src/PerfView/Properties/PerfView.manifest

```diff
@@ -0,0 +1,81 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
❔ Does this need to match the application name? If not, this entire file can be replaced by:
https://github.com/dotnet/roslyn/blob/e704ca635bd6de70a0250e34c4567c7a28fa9f6d/eng/config/app.manifest

Reply:
Ah, looks like support for Roslyn is implemented here:

https://github.com/dotnet/roslyn/blob/4758a7be7e9e59204bcee81eccd8309a5c04d393/eng/targets/Imports.targets#L322-L367

Reply:
yes but only for .NET 4.7.2 
Condition="'$(OutputType)' == 'Exe' And '$(TargetFramework)' == 'net472'">

Reply:
I do not worry about the assembly version in the manifest because no one would want to reference PerfView from unmanaged code via a manifest reference. I think we can keep it to 1.0.0.0 without any issues.